test/rules/named-spy.js

```diff
@@ -32,6 +32,11 @@ eslintTester.run('named-spy', rule, {
       '};'
     ]),
     linesToCode([
+      'someObject = {',
```

Comment:
some extra tests for coverage would be really nice

Reply:
I'll add a negative test case. Can you think of some other test which should be added?